City-of-Helsinki/drupal-helfi-sote
# Changed changed field to content_translation_changed.

How to test: 
- Checkout this branch and do `make drush-cim drush-cr`
- Login and go to https://helfi-sote.docker.so/fi/admin/content/integrations/tpr-unit
- Check that the view works
